parm/config/gfs/config.com

```diff
@@ -84,6 +84,7 @@ declare -rx COM_OCEAN_INPUT_TMPL=${COM_BASE}'/model/ocean/input'
 declare -rx COM_OCEAN_ANALYSIS_TMPL=${COM_BASE}'/analysis/ocean'
 declare -rx COM_OCEAN_BMATRIX_TMPL=${COM_BASE}'/bmatrix/ocean'
 declare -rx COM_OCEAN_NETCDF_TMPL=${COM_BASE}'/products/ocean/netcdf'
+declare -rx COM_OCEAN_NETCDF_GRID_TMPL=${COM_OCEAN_NETCDF_TMPL}'/${GRID}'
```

Comment:
This is being added here to serve as a `COMIN` for archive, but what job is placing files in this location? It can't be a `COMIN` unless another job is first using it as a `COMOUT`.

Reply:
It appears it comes from the extract vars job, but that job does not use this template. It should be updated to use the same template.

Reply:
```suggestion

```